tox.ini

```diff
@@ -94,6 +94,9 @@ setenv =
   ; i.e: CONTRIB_REPO_SHA=dde62cebffe519c35875af6d06fae053b3be65ec tox -e <env to test>
   CONTRIB_REPO_SHA={env:CONTRIB_REPO_SHA:"main"}
   CONTRIB_REPO="git+https://github.com/open-telemetry/opentelemetry-python-contrib.git@{env:CONTRIB_REPO_SHA}"
+  ; Enable more git debugging. With this change CI passes again (which is unexplained). See #3235.
+  GIT_TRACE=1
+  GIT_TRACE_SETUP=true
```

Comment:
does this spam the output logs?

Reply:
Yes, it does, removing. We can try with less intrusive things like showing pip version (which gives us some useful information at least).